kafka-testing/src/test/java/org/jsmart/zerocode/integration/tests/kafka/protobuf/KafkaProtobufTest.java

```diff
@@ -0,0 +1,19 @@
+package org.jsmart.zerocode.integration.tests.kafka.protobuf;
+
+import org.jsmart.zerocode.core.domain.Scenario;
+import org.jsmart.zerocode.core.domain.TargetEnv;
+import org.jsmart.zerocode.core.runner.ZeroCodeUnitRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@TargetEnv("kafka_servers/kafka_test_server_protobuf.properties")
+@RunWith(ZeroCodeUnitRunner.class)
+public class KafkaProtobufTest {
```

Comment:
Minor comment,
`KafkaProtobufTest.class` should go here into the
`org.jsmart.zerocode.integration.tests.kafka.KafkaSuite` for the CI job to pick it as part of regression. ✌️

Reply:
I changed it to use a generic bytearray serializer and deserializer please take a look